scripts/scripts.js

```diff
@@ -292,15 +292,19 @@ async function detectSidebar(main) {
     for (let i = 0; i < sidebarOffset - 1; i += 1) {
       main.children[i].classList.add('no-sidebar');
     }
-
-    sidebar.querySelectorAll('h3').forEach((header) => {
-      const headerContent = header.textContent;
-      header.textContent = '';
-      header.append(span(headerContent));
-    });
   }
 }
 
+async function h3Styling(main) {
```

Comment:
Does it have to be async?

Reply:
Yes, first i tried without async, threw error as the elements were not available on the page.

Reply:
moving it to decorate main and removing async works fine. Updated the code.